src/nxdoc/web-ui/web-ui-upgrade-notes/web-ui-11-1-upgrade-notes.md

```diff
@@ -79,5 +85,95 @@ Removed variables can be safely deleted from your themes or elements. Deprecated
+
+#### Nuxeo DAM addon removal
+
+Nuxeo DAM no longer exists as an addon and its contributions are now default on Web UI. If you're using Nuxeo Studio, you will see the Nuxeo DAM in the removed addons list when upgrading your project through the application definition page.
+
+#### Spreadsheet addon removal
+
+Spreadsheet addon is now loaded by default but the button contribution is disabled. Users can rely on Studio Designer to re-enable the `spreadSheet` *button* in the `RESULTS_ACTIONS` slot (see [WEBUI-90](https://jira.nuxeo.com/browse/WEBUI-90)).
+
+{{!--     ### nx_asset ###
+    path: /default-domain/workspaces/Product Management/Documentation/Documentation Screenshots/NXDOC/Master/Web UI 11.x upgrade notes/Spreadsheet package enable button in Designer
+    name: spreadsheet-package-enable-button-designer.png
+    1.1.3#screenshot#up_to_date
+--}}
+![Spreadsheet package enable button in Designer](nx_asset://b8fd28dd-0272-43b1-a083-dfde295c312b ?w=650,border=true)
+
+### BROWSER_ACTIONS slot removal
+
+The `BROWSER_ACTIONS` nuxeo slot was removed under [NXP-26184](https://jira.nuxeo.com/browse/NXP-26184). It was already deprecated since Web UI 0.9 and had no known usage. It was replaced by the `RESULTS_SELECTION_ACTIONS` slot.
+
+### Forked nuxeo-document-content might lose selection actions
+
+After [NXP-25345](https://jira.nuxeo.com/browse/NXP-25345), Nuxeo Web UI introduced the ability to override selection actions. Elements that were forked from an older version of `nuxeo-document-content` and that override the `selectionActions` native slot with new content will be missing the contributions to the `RESULTS_SELECTION_ACTIONS` nuxeo slot. This can be rectified by adding the desired actions to the new slot, and by deleting the following piece of code:
+
+```
+<div slot=“selectionActions”>
+  <nuxeo-slot slot=“BROWSE_ACTIONS” model=“[[browseActionContext]]“></nuxeo-slot>
+</div>
+```
+
+### Picture document page being displayed on doctypes using the Picture facet
+
+Since [NXP-25740](https://jira.nuxeo.com/browse/NXP-25740), the `nuxeo-picture-document-page` is now displayed for documents with `Picture` **facet** instead of documents with `Picture` **type**. This means that on migrated projects, the `nuxeo-picture-document-page` might be displayed on documents where it was not expected. If this is not desirable, the contribution can simply be overridden to only display the page for documents with `Picture` **type**:
+
+```
+<nuxeo-slot-content name="pictureDocumentViewPage" slot="DOCUMENT_VIEWS_PAGES" order="5">
+  <template>
+    <nuxeo-filter document="[[document]]" type="Picture">
+      <template>
+        <nuxeo-picture-document-page name="view" document="[[document]]"></nuxeo-picture-document-page>
+      </template>
+    </nuxeo-filter>
+  </template>
+</nuxeo-slot-content>
+
+```
+
+### Deprecate usage of nuxeo-document-history
```

Comment:
I think we should group these in a section for deprecations. Something like:

```
### Deprecations
...
#### Deprecate usage of nuxeo-document-history
...
#### Deprecated css variables
...
```

Reply:
I think we are also missing the deprecation of the following CSS variables:
--nuxeo-document-creation-form-icon-width
--nuxeo-document-creation-form-icon-height
in favor of the --nuxeo-document-create-selection-icon mixin (see [NXP-27037](https://jira.nuxeo.com/browse/NXP-27037))

Reply:
Also missing the following deprecations:
- [ELEMENTS-1012](https://jira.nuxeo.com/browse/ELEMENTS-1012): use `name` instead of `slot` for nuxeo slots
- [ELEMENTS-1153](https://jira.nuxeo.com/browse/ELEMENTS-1153): deprecated `login` test helper in favor or `MockClient`

Reply:
> I think we are also missing the deprecation of the following CSS variables:
> --nuxeo-document-creation-form-icon-width
> --nuxeo-document-creation-form-icon-height
> in favor of the --nuxeo-document-create-selection-icon mixin (see [NXP-27037](https://jira.nuxeo.com/browse/NXP-27037))

How did you find out the deprecation for these CSS variables?

Reply:
Well, in fact how did you find all these deprecations?

Reply:
Did a search on VSCode for "deprecated" on Nuxeo Web UI and Nuxeo Elements projects. 😛

Reply:
I'll add them to the docs. Thanks

Reply:
Ahhhh, I know why I missed those 🤦 I'm still not used to search on both repos (web-ui and elements)😛